AnnoDesigner.Core/Models/GridDirection.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AnnoDesigner.Core.Models
+{
+    public enum GridDirection
+    {
+        Up,
```

Comment:
Is it smart to add a value `Undefined`?

Reply:
I do not think that would be a good thing, as the direction has 4 sides to go, and he is rotating them that way

Reply:
Yeah, what would it mean for a building to have `Undefined` GridDirection? Each building can be rotated yet its blocked would not rotate?

Reply:
Ok I understand your thinking. It was just an idea 😀